README.md

```diff
@@ -104,3 +104,7 @@ Your obligations (and my wish) are:
  - If you modify the SpinalHDL core (the compiler itself), please, share your improvements.
 
 Also, SpinalHDL is provided "as is", without warranty of any kind.
+
```

Comment:
Wouldn't this note be better placed in the RTD repo? As far as I can tell you are not translating SpinalHDL itself?

Reply:
It's a good point.